wondrous-app/components/Common/SidebarMainPods/index.tsx

```diff
@@ -22,7 +37,7 @@ const PodsIconButton = () => {
 
   return (
     <>
-      <PodModal open={openPodModal} handleClose={() => setOpenPodModal(false)} />
+      <PodModal open={openPodModal} handleClose={() => setOpenPodModal(false)} pods={useGetUserPods()} />
```

Comment:
why do we need to  do this here?
useGetUserPods()

Reply:
seems like an uncessary call

Reply:
I added pods to the parameters of PodModal so it can be reused. Before, the PodModal only shows the user's pods. To fix that issue, I decoupled the data so we can reuse the PodModal in the project page to show the org's pods.

This is the related commit https://github.com/wondrous-dev/wondrous-frontend/pull/1284/commits/9c7c71a4ca72226f7ca17f64e7df37e8a667674a

Reply:
oh i see, i think we actually don't need that modal! i think we either dont need the show all button or it would be better to use the <MoreInfoModal> in the wrapper

<img width="694" alt="Screen Shot 2022-12-01 at 2 14 01 PM" src="https://user-images.githubusercontent.com/19996922/205170350-b57a07a3-072e-495e-b897-0f83be0f6d3a.png">

Reply:
the point is to save a query

Reply:
Ah right! I'll just use that.